src/desktop/src/ui/components/Chart.js

```diff
@@ -54,7 +54,7 @@ class Chart extends PureComponent {
             const date = new Date(props.payload[0].payload.time * 1000);
             return (
                 <p className={css.label}>
-                    {format(date, 'DD.MM.YY HH:mm')}
+                    {formatTimeAs.dayMonthYearHoursMinutes(navigator.language, detectedTimezone, date)}
```

Comment:
Should we use system language (`navigator.language`) or the user chosen wallet language 🤔

Reply:
I used the system language so that the functionality would be the same as mobile (https://github.com/iotaledger/trinity-wallet/pull/434/files#diff-cd6bfd84c7bace320f0bf28eab3ba245R8). `moment` supports more locales than we do, so if the user uses English because their native language isn't supported by Trinity, it will still use their native date/time format to match how the rest of their device will format dates/times.